src/FsCodec.SystemTextJson/Interop.fs

```diff
@@ -0,0 +1,56 @@
+namespace FsCodec.SystemTextJson
+
+open System.Runtime.CompilerServices
+open System.Text.Json
+
+[<Extension>]
+type InteropExtensions =
+    static member private noOverescapingOptions =
+        System.Text.Json.JsonSerializerOptions(Encoder = System.Text.Encodings.Web.JavaScriptEncoder.UnsafeRelaxedJsonEscaping)
+    static member private Adapt<'From, 'To, 'Event, 'Context>
+        (   native : FsCodec.IEventCodec<'Event, 'From, 'Context>,
+            up : 'From -> 'To,
+            down : 'To -> 'From) : FsCodec.IEventCodec<'Event, 'To, 'Context> =
+
+        { new FsCodec.IEventCodec<'Event, 'To, 'Context> with
+            member __.Encode(context, event) =
+                let encoded = native.Encode(context, event)
+                { new FsCodec.IEventData<_> with
+                    member __.EventType = encoded.EventType
+                    member __.Data = up encoded.Data
+                    member __.Meta = up encoded.Meta
+                    member __.EventId = encoded.EventId
+                    member __.CorrelationId = encoded.CorrelationId
+                    member __.CausationId = encoded.CausationId
+                    member __.Timestamp = encoded.Timestamp }
+
+            member __.TryDecode encoded =
+                let mapped =
+                    { new FsCodec.ITimelineEvent<_> with
+                        member __.Index = encoded.Index
+                        member __.IsUnfold = encoded.IsUnfold
+                        member __.Context = encoded.Context
+                        member __.EventType = encoded.EventType
+                        member __.Data = down encoded.Data
+                        member __.Meta = down encoded.Meta
+                        member __.EventId = encoded.EventId
+                        member __.CorrelationId = encoded.CorrelationId
+                        member __.CausationId = encoded.CausationId
+                        member __.Timestamp = encoded.Timestamp }
+                native.TryDecode mapped }
+
+    static member private MapFrom(x : byte[]) : JsonElement =
```

Comment:
@ylibrach the tests are failing - help!

Reply:
STJ not working on net461 is the only remaining issue; will mull it over and fix